Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging_test.py

```diff
@@ -51,6 +52,14 @@ def test_sign_and_encrypt(mocker):
            'Content-Transfer-Encoding: base64' in sign_encrypt
 
 
+def test_decode_using_chardet():
+
+    decrypted_text_bytes = b'Za\xbf\xf3\xb3\xe6 g\xea\xb6l\xb1 ja\xbc\xf1'
+    out, error = decode_str_using_chardet(decrypted_text_bytes)
```

Comment:
I can not reproduce this case because the encrypted file of the customer is signed. When I encrypt this kind of chars it does not fails on decoding (the problem of the customer is that the file was encrypt with encoding that we didnt use on the decrypt.